tests/e2e/specs/shopper/cart-products.test.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { SIMPLE_PRODUCT_NAME } from '../../../utils/constants';
+
+const block = {
+	name: 'Cart',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Cart → Can remove product', () => {
+	beforeAll( async () => {
+		await shopper.block.emptyCart();
+	} );
+
+	it( 'Can remove product from cart', async () => {
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
```

Comment:
I didn't know this existed, nice one, this makes adding items to cart a one liner!